subprojects/packagefiles/blake3/meson.build

```diff
@@ -19,12 +19,12 @@ simd_avx2 = '''
 int main (int argc, char *argv[]) { __m256i add0 = _mm256_set_epi32(7, 6, 5, 4, 3, 2, 1, 0); return 0; }
 '''
 
-simd_avx512 = '''
+simd_avx512vl = '''
 #if defined(_MSC_VER)
 #include <intrin.h>
 #endif
 #include <immintrin.h>
-int main (int argc, char *argv[]) { __m256i add0 = _mm256_setr_epi64x(0, 1, 2, 3); return 0; }
+int main (int argc, char *argv[]) { __m128i v = _mm_ror_epi32(_mm_set_epi32(3, 2, 1, 0), 5); return 0; }
```

Comment:
```c
const __m512i shifts  = packed_qword(0x3036242a1016040alu); // 48, 54, 36, 42, 16, 22, 4, 10
const __m512i indices = _mm512_multishift_epi64_epi8(shifts, in);
```
You could use this instead. this is `avx512vl` only

Reply:
Accoriding to these docs, `_mm512_multishift_epi64_epi8` is AVX512VBMI: https://www.intel.com/content/www/us/en/docs/cpp-compiler/developer-guide-reference/2021-8/intrinsics-for-bit-manipulation-operations-001.html

Reply:
`_mm256_lzcnt_epi32` ?

Reply:
That also needs AVX512CD. Is there any disadvantage in `_mm_ror_epi32`? It does `vprord` in a similar way that the blake3 code does.

Reply:
i think is badly detected. also that one is AVX512VL and CD

Reply:
This says that `_mm_ror_epi32` is VL and F, which is just what we need: https://www.intel.com/content/www/us/en/docs/cpp-compiler/developer-guide-reference/2021-8/intrinsics-for-shift-operations.html